src/sql/src/plan/error.rs

```diff
@@ -474,6 +480,13 @@ impl fmt::Display for PlanError {
             Self::DropLastSubsource { source } => write!(f, "SOURCE {} must retain at least one non-progress subsource", source.quoted()),
             Self::DropProgressCollection { progress_collection, source: _} => write!(f, "SOURCE {} is a progress collection and cannot be dropped independently of its primary source", progress_collection.quoted()),
             Self::DropNonSubsource { non_subsource, source} => write!(f, "SOURCE {} is a not a subsource of {}", non_subsource.quoted(), source.quoted()),
+            Self::DependentObjectsStillExist {object_type, object_name, dependents} => {
+                let reason = match &dependents[..] {
+                    [] => " because other objects depend on it".to_string(),
+                    dependents => format!(": still depended upon by {}", dependents.join(", ")),
+                };
+                write!(f, "cannot drop {object_type} {object_name}{reason}")
```

Comment:
`object_name` and all entries in `dependents` should be quoted

Reply:
Done